Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -8,17 +8,23 @@
 
 
 // Controller
-#import "WMFArticleFetcher.h"
 #import "WMFArticleViewController.h"
 #import "WebViewController.h"
 #import "UIViewController+WMFStoryboardUtilities.h"
 #import "WMFSaveButtonController.h"
 #import "WMFPreviewController.h"
 #import "WMFArticleContainerViewController_Transitioning.h"
+#import "WMFArticleHeaderImageGalleryViewController.h"
```

Comment:
That said, the class still weighs in at a not-totally-unreasonable ~550 LOC (w/ plenty of multi-line assignments and things).  Compared to the OG `WebViewController` at over 1900 LOC :scream: